apps/emqx_connector/test/emqx_connector_api_SUITE.erl

```diff
@@ -92,9 +93,11 @@ init_per_suite(Config) ->
     ok = emqx_common_test_helpers:load_config(emqx_connector_schema, <<"connectors: {}">>),
     ok = emqx_common_test_helpers:load_config(emqx_rule_engine_schema, <<"rule_engine {rules {}}">>),
     ok = emqx_common_test_helpers:load_config(emqx_bridge_schema, ?BRIDGE_CONF_DEFAULT),
+    emqx_dashboard_admin:add_user(<<"connector_admin">>, <<"public">>, <<"administrator">>),
     Config.
 
 end_per_suite(_Config) ->
+    mnesia:clear_table(?ADMIN),
```

Comment:
Q: what are the changes in this file needed for?

I tried to run only this suite locally, and apparently all tests executed successfully. :thinking:

Reply:
This was copied out of the dashboard tests. It seemed like this was "the way" to make sure the users added to the dashboard admin were cleared. Given that I've changed the name of the user to be specific to the connector it's not going to be a problem with tests forming temporal dependencies but put it in there for "completeness", but if it isn't necessary can remove it.

Reply:
mb. we may use `emqx_dashboard_admin:remove_user(<<"connector_admin">>)` for explicitness.

Reply:
I just realized these changes should not be in this commit. They were left over from when I realized the problem with the tests being reliant on the dashboard tests. I made a separate commit to fix things but didn't remove these changes from this branch. Will make a different self-contained commit for the suggested change.